packages/PostRector/Rector/ClassRenamingPostRector.php

```diff
@@ -65,6 +64,13 @@ public function enterNode(Node $node): ?Node
         $scope = $node->getAttribute(AttributeKey::SCOPE);
         $result = $this->classRenamer->renameNode($node, $oldToNewClasses, $scope);
 
+        if (! $result instanceof Name && ! $node instanceof FullyQualified) {
+            $phpAttributeName = $node->getAttribute(AttributeKey::PHP_ATTRIBUTE_NAME);
+            if (is_string($phpAttributeName)) {
+                return $this->classRenamer->renameNode(new FullyQualified($phpAttributeName, $node->getAttributes()), $oldToNewClasses, $scope);
+            }
+        }
```

Comment:
This is the trick 👍 , on `ClassRenamingPostRector`, verify that `Name` is not `FullyQualified`, which:

```php
$this->phpAttributeGroupFactory->create()
```

can return a `Name` instead of `FullyQualified` with bring original attribute name to be renamed, then use it to be returned so it can be processed on next loop on `FileProcessor`.